### ISSUE TITLE
Windows Error Message if started over Shortcut

### ISSUE THREAD

**Issue by kai-tub** [Cimbali/pympress#150]
# Thanks
First of all, thank you for providing such a cool tool! I love that I can use it on all of my machines!

# Environment
- Windows10

# Pympress
- 1.4.1
- Installation method: Chocolatey

# Steps to reproduce
Opening pympress over the start menu.

# Problem
Multiple error messages at startup:

> The procedure entry point DllMain could not be located in the dynamic link library C:\Program Files(x86)\pympress\lib\gdk-pixbuf-2.0\2.10.0\loaders\libpixbufloader-svg.dll

> The procedure entry point DllMain could not be located in the dynamic link library C:\Progam Files\Inkscape\librsvg-2.2dll

which is repeated about 10x.

Afterwards, the program seems to work normally. If I start pympress directly over the terminal everything works fine.

# pympress.log
INFO:__main__:Pympress: 1.4.1 ; Python: 3.7.4 ; OS: Windows 10 ; Gtk 3.24.10 ; GLib 2.60.4 ; Poppler 0.78.0 cairo ; Cairo 1.16.0 , pycairo 1.18.1 ; Media: GtkImage gif player, VLC 3.0.8 Vetinari

**Comment by Cimbali** [comment]
I’m not sure I understand where those messages appear, if you say you’re starting pympress from the start menu.

Are they error popups, like a small window with a X on red background and an “OK” button?
Or are the errors showing in some kind of console somewhere?

**Comment by kai-tub** [comment]
Sorry for not being very clear.
I've attached some pictures and hopefully, they make it more clear. :)

![capture00](https://user-images.githubusercontent.com/46302524/77237866-1c44e900-6bcc-11ea-964a-a8af6b639536.png)
![capture01](https://user-images.githubusercontent.com/46302524/77237868-1fd87000-6bcc-11ea-8d7a-6b6e4d9df5eb.png)
![capture02](https://user-images.githubusercontent.com/46302524/77237871-22d36080-6bcc-11ea-905a-d0bde6857222.png)

**Comment by Cimbali** [comment]
Thanks a lot for the details @kai-tub ! That seems pretty annoying indeed.

**Comment by kai-tub** [comment]
I am happy to help as much as possible.
And I think I have a possible fix.
If I remove Inkscape from my windows path, the error doesn't occur. But I need Inkscape in my path for my presentation, otherwise, I couldn't import a vector graphic file.

**Comment by Cimbali** [comment]
So this seems to be due to pympress and inkscape being shipepd with different versions of the same libraries.

I’m not sure there’s a lot that can be done about this, but can you try adding pympress to the path as well? In such a way that the path to pympress appears before the path of Inkscape.

**Comment by Cimbali** [comment]
Also, are your inkscape and pympress of the same bitness (32bit or 64bit)? If they’re not that could be the problem (and why I can’t reproduce the issue).

You can see how to check this information [in this tutorial](https://www.ghacks.net/2016/12/09/find-out-if-a-windows-program-is-32-bit-or-64-bit/). Basically:
- Open the Task Manager (Ctrl + Shift + Escape)
- If you don’t see several tabs, click “More Details” 
- go to the “Details” tab
- Add the “plaform” column. To do this:
    - Right click the column header
    - Click “Select columns”
    - Tick the “Platform” option and click the OK button
- Now you can inkscape (or pympress) and check on its line in the details tab, whether it says “32 bits” or “64 bits” in the platform column.

**Comment by kai-tub** [comment]
A couple of things I've tested:
1. Yes, I had 32bit pympress and 64bit inkscape + an old version (chocolatey install is 1.4.1)
2. Removed 32bit pympress and installed 64bit pympress version 1.5.2
    - Problem maintains
3. Moved `inkscape` from user path to system path (before and after pympress)
    - Both seem to work
4. Moved pympress to user path with inkscape before and after
    - Both *do not* work

So my current solution is to move the Inkscape path to the system path.
I can also upload the logs if it helps. :)
